hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/SimpleLoadBalancer.java

```diff
@@ -136,7 +136,7 @@ public void onConfigurationChange(Configuration conf) {
     super.setConf(conf);
     LOG.info("Update configuration of SimpleLoadBalancer, previous slop is "
             + originSlop + ", current slop is " + slop + "previous overallSlop is" +
-            originOverallSlop + ", current overallSlop is " + originOverallSlop);
+            originOverallSlop + ", current overallSlop is " + overallSlop);
```

Comment:
nit: can you use parameterized `{}` log syntax?
```
    LOG.info(
      "Update configuration of SimpleLoadBalancer, previous slop is {}, current slop is {} previous overallSlop is {}, current overallSlop is {}",
      originSlop, slop, originOverallSlop, overallSlop);
```

Reply:
Do it now.